src/components/EventCard.tsx

```diff
@@ -44,7 +44,7 @@ const EventsCardComponent: React.FC<EventsCardProps> = ({
   to,
 }) => {
   return (
-    <EventsCard cover={<img alt="example" src={src} />}>
+    <EventsCard cover={<img alt="example" src={src || DEFAULT_IMAGE} />}>
```

Comment:
Do wanna apply default images only on the frontend, or should we create announcements with a default image on the backend if the imageSrc wasn't specified in the POST request?

In another PR review of your branch, I think I mentioned that the backend could do it, but I think I would be better to not manipulate the announcement data, and keep the default image application as a UI feature only. 

Wonder what your thoughts are @jackblanc

Reply:
IMO its better to have this on the frontend, but I'll refrain from merging until we have a consensus

Reply:
I think the default image should be client-side, as long as we always intend for it to be the same image default for all occurrences. Makes it easier to change I suppose.